packages/realm-network-transport/src/react-native/index.ts

```diff
@@ -0,0 +1,30 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2021 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+export * from "../index";
+
+import { DefaultNetworkTransport } from "../DefaultNetworkTransport";
+import { AbortController, Fetch } from "../types";
+
+DefaultNetworkTransport.fetch = globalThis.fetch.bind(globalThis) as Fetch;
+DefaultNetworkTransport.AbortController = globalThis.AbortController as AbortController;
+// Setting this non-standard option to enable text streaming
+// See https://github.com/react-native-community/fetch#enable-text-streaming
+DefaultNetworkTransport.extraFetchOptions = {
+  reactNative: { textStreaming: true },
```

Comment:
See https://github.com/react-native-community/fetch#enable-text-streaming for details on why we're adding this.

Reply:
Please add it as a comment

Reply:
It's already there - two lines above the line I commented on here :)